vms/platformvm/config/execution_config.go

```diff
@@ -36,6 +36,7 @@ type ExecutionConfig struct {
 	ChainDBCacheSize             int            `json:"chain-db-cache-size"`
 	BlockIDCacheSize             int            `json:"block-id-cache-size"`
 	FxOwnerCacheSize             int            `json:"fx-owner-cache-size"`
+	ChainIDAndAddrCacheSize      int            `json:"chain-id-and-addr-cache-size"`
```

Comment:
Do we not have a more concise name for this? `SubnetManager` or something? `ChainIDAndAddr` feels pretty amorphous.

Reply:
was mostly taking the queue from `fx-owner-cache-size`, updated to `subnet-manager-cache-size`